packages/invoices/src/InboundShipment/ListView/ListView.tsx

```diff
@@ -35,7 +33,7 @@ export const InboundListView: FC = () => {
 
   const columns = useColumns<InboundRowFragment>(
     [
-      [getNameAndColorColumn(), { setter: () => {} }],
+      [getNameAndColorColumn(), { setter: onUpdate }],
```

Comment:
and this is essentially what I did for Request/Response/Inbound/Outbound

- Move modal to AppBarButtons
- Make `useUpdateX` hook
- Use hook as setter here